commons/src/test/scala/dagr/commons/util/StringUtilTest.scala

```diff
@@ -87,4 +87,26 @@ class StringUtilTest extends UnitSpec {
     StringUtil.camelToGnu("A") should be ("a")
     StringUtil.camelToGnu("b") should be ("b")
   }
+
+  "StringUtil.addSpacesToCamelCase" should "correctly handle simple multi-word variable names" in {
+    StringUtil.addSpacesToCamelCase("fooBar") should be ("foo Bar")
+    StringUtil.addSpacesToCamelCase("oneTwoThreeFourFiveSixSeven") should be ("one Two Three Four Five Six Seven")
+  }
+
+  it should "not modify single word variable names" in {
+    StringUtil.addSpacesToCamelCase("foo") should be ("foo")
+  }
+
+  it should "not add a space when the word starts with a caps" in {
+    StringUtil.addSpacesToCamelCase("BigBad") should be ("Big Bad")
+  }
+
+  it should "insert spaces between adjacent capital letters" in {
+    StringUtil.addSpacesToCamelCase("TisASillyName") should be ("Tis A Silly Name")
+  }
+
+  it should "handle single letter names" in {
+    StringUtil.addSpacesToCamelCase("A") should be ("A")
+    StringUtil.addSpacesToCamelCase("b") should be ("b")
+  }
```

Comment:
Only question is do you want to add a case for multiple upper-case letters in a row?  E.g. `thisIsAThing` vs. `getSAMString`.